src/main/java/com/ibm/eventstreams/connect/mqsource/processor/JmsToKafkaHeaderConverter.java

```diff
@@ -0,0 +1,51 @@
+package com.ibm.eventstreams.connect.mqsource.processor;
```

Comment:
Add copyright statement to match the other source files.

Reply:
done